src/Park.js

```diff
@@ -18,16 +18,23 @@ class Park extends Component {
     event.preventDefault();
     this.props.removeCard();
     this.setState({
-      displayFull: false
+      displayFull: false,
+      annualVisitors: 0
     });
   }
 
   updateParkCodes = (storageKey, newArray) => {
     this.props.updateParkCodes(storageKey, newArray);
   }
 
+  commaSeparateAnnualVisitors = () => {
+    let visitors = parseInt(this.props.selectedPark.annualVisitors);
```

Comment:
Is this line necessary? I thought annual visitors was stored as a number in the dataset so we may not need this step.